src/Microsoft.IdentityModel.Clients.ActiveDirectory/Internal/Flows/AcquireTokenInteractiveHandler.cs

```diff
@@ -126,12 +126,13 @@ private static string ReplaceHost(string original, string newHost)
             await this.AcquireAuthorizationAsync().ConfigureAwait(false);
             this.VerifyAuthorizationResult();
 
-            if (!string.IsNullOrEmpty(authorizationResult.CloudInstanceHost))
+            //this code is commented out to disable sovereign cloud flow
+/*            if (!string.IsNullOrEmpty(authorizationResult.CloudInstanceHost))
             {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://iddp-sonarqube:9000/coding_rules#rule_key=csharpsquid%3AS125)